package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "victory-line",
-  "version": "0.7.0",
+  "version": "0.8.0",
```

Comment:
NIT: no manual version bumps. We've got a script for that https://github.com/FormidableLabs/builder-victory-component/blob/master/dev/DEVELOPMENT.md#releases